hacspec/src/math_util.rs

```diff
@@ -385,3 +387,291 @@ pub fn ct_div<T: Integer + Copy>(a: T, d: T) -> (T, T) {
     }
     (q, r)
 }
+/// makes poly to an element of R_modulo \ irr
+pub fn R(irr:&Seq<i128>,poly:&Seq<i128>,modulo:i128) -> Seq<i128>{
```

Comment:
Yes, I am going to mention it.
It would be possible to use Seq<u128> but in case of NTRU prime it will increase the runtime. Because you would need to convert from Seq<u128> to Seq<i128> often.

Reply:
Yeah but if it's not too slow we should do it :)
Proper types are more important here than speed (as long as its reasonable)

Reply:
Okay :)